coordinator/src/main/java/com/tencent/rss/coordinator/AccessCandidatesChecker.java

```diff
@@ -45,12 +51,21 @@
 
   private final AtomicReference<Set<String>> candidates = new AtomicReference<>();
   private final AtomicLong lastCandidatesUpdateMS = new AtomicLong(0L);
-  private final String path;
+  private final Path path;
   private final ScheduledExecutorService updateAccessCandidatesSES;
+  private final FileSystem fileSystem;
 
-  public AccessCandidatesChecker(AccessManager accessManager) {
+  public AccessCandidatesChecker(AccessManager accessManager) throws Exception {
     CoordinatorConf conf = accessManager.getCoordinatorConf();
-    this.path = conf.get(CoordinatorConf.COORDINATOR_ACCESS_CANDIDATES_PATH);
+    String pathStr = conf.get(CoordinatorConf.COORDINATOR_ACCESS_CANDIDATES_PATH);
```

Comment:
Is it necessary to add a check to validate path?

Reply:
It's a good idea to check file existence when the Coordinator starts.